blocks/editable/index.js

```diff
@@ -487,16 +465,12 @@ export default class Editable extends Component {
 	}
 
 	setContent( content ) {
-		if ( ! content ) {
-			content = '';
-		}
-
-		content = renderToString( content );
-		this.editor.setContent( content, { format: 'raw' } );
+		this.editor.setContent( content || '' );
```

Comment:
Do you think we could drop this method then?

Reply:
no strong opinion here, but maybe we should keep it to avoid handling the default value in the different places we call this.

Reply:
Yeah, actually makes sense if we keep `getContent` too.